catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/MetacardImpl.java

```diff
@@ -843,15 +858,60 @@ private void readObject(ObjectInputStream stream) throws IOException, ClassNotFo
 
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Oh and looks like phil beat me to this comment above.
https://github.com/codice/ddf/pull/1379#issuecomment-256426288